ovos_plugin_common_play/ocp/player.py

```diff
@@ -669,12 +832,22 @@ def handle_playlist_clear_request(self, message):
 
     # audio ducking
     def handle_duck_request(self, message):
+        """
+        Pause audio on 'recognizer_loop:record_begin'
+        @param message: Message associated with event
+        """
         if self.state == PlayerState.PLAYING:
             self.pause()
+            self._paused_on_duck = True
```

Comment:
should a `DuckingState` be introduced? could be useful if we want to signal it in the GUI/LED events etc

Reply:
I'm not sure the state would need to be more than a boolean.. Either we were playing and are now ducked, or we don't care about the ducking state, right?

Reply:
i meant as a generic property to be exposed via messagebus and MPRIS, thinking about 3rd party consumers of this info

Reply:
I think it would make more sense to put that in the `audio` module since we could then also resolve the TODO of waiting for audio output to end. It also would make it available to installations/devices that don't implement OCP media playback.

Reply:
not sure about that.... because OCP can be used standalone too, it is not a audio module property either, audio delegates playback to OCP so OCP should be the one reporting the end of playback

Reply:
dont mean this as a PR blocker, it can be done some other time, it's quite unimportant and out of scope! want those fixes and unittests in ASAP :)